AddRoutineView.swift

```diff
@@ -0,0 +1,163 @@
+    @State var dates: String = ""
+    @State var streakLength: Int = 0
+    
+    @StateObject var vm = ViewModel()
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Text("OnTrack")
+                    .font(.largeTitle)
+                    .bold()
+                    .padding(.leading)
+                
+                Spacer()
+                
+                
+            }
+            
+            Text("Add Routine")
+                .padding(.vertical)
+                .font(.largeTitle)
+               .italic()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add an ID", text: $id)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a title", text: $title)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            
+
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a description", text: $description)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 70)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a category", text: $category)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a frequency", value: $frequency, format: .number)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
```

Comment:
I forgot to clarify this in the google doc, but in a prior meeting we discussed that the user can select fixed frequencies (e.g. daily, weekly, monthly, etc.)

The `Routine` struct handles frequency as a number of seconds, and we definitely don't want to make the user type in "86400" to specify a daily task.